service-netty/src/main/java/info/bitrich/xchangestream/service/netty/WebSocketClientHandler.java

```diff
@@ -109,6 +109,7 @@ private void dealWithContinuation(ContinuationWebSocketFrame frame) {
 
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
+        cause.printStackTrace();
```

Comment:
The right place to do this is by including it in the `LOG.error()` below:
```
LOG.error("WebSocket client encountered exception. Closing", cause);
```
... since `e.printStackTrace()` doesn't go through the "proper" log handler route, just direct to stdout, which will cause problems for people with special log handling requirements.

However, I'm pretty sure that if you look at the history of this code, you'll find that the above is exactly what it used to do, but we suppressed the actual stack trace since it ended up being too noisy. Unexpected errors on websockets turn out to happen an awful lot (the internet isn't reliable) it was spamming the logs too much.

Generally we try and catch exceptions in more case-specific ways.  Was this here to catch something in particular?